.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: swift
-osx_image: xcode11.3
+osx_image: xcode11.4
```

Comment:
Should we (can we) specify Xcode 11.4.1?

Reply:
Looks like `xcode11.4` isn't actually officially documented yet on https://docs.travis-ci.com/user/reference/osx/#macos-version, but they do say that `xcode11.3` was actually tied to Xcode 11.3.1, so sounds like they include patch releases automatically